src/rard/research/views/mention.py

```diff
@@ -178,8 +178,19 @@ def get(self, request, *args, **kwargs):
             if not model_name:
                 model_name = next(k for k, value in dd.items() if value == o.__class__)
                 model_name_cache[o.__class__] = model_name
-
-            ajax_data.append({"id": o.pk, "target": model_name, "value": str(o)})
+            if model_name == "bibliographyitem":
+                ajax_data.append(
+                    {
+                        "id": o.pk,
+                        "target": model_name,
+                        "value": str(o),
+                        "citation": BibliographyItem.objects.get(
+                            pk=o.pk
+                        ).mention_citation(),
+                    }
+                )
+            else:
+                ajax_data.append({"id": o.pk, "target": model_name, "value": str(o)})
```

Comment:
A more resilient approach might be:
```suggestion
            citation = o.mention_citation() if hasattr(o,"mention_citation") else str(o)
            ajax_data.append(
                {
                    "id": o.pk,
                    "target": model_name,
                    "value": str(o),
                    "citation": citation,
                }
            )
```

Reply:
That'd leave you the flexibility to create custom citations for other models by adding a mention_citation method if the need arises